pkg/snmp/traps/config.go

```diff
@@ -18,13 +21,27 @@ func IsEnabled() bool {
 	return config.Datadog.GetBool("snmp_traps_enabled")
 }
 
+// UserV3 contains the definition of one SNMPv3 user with its username and its auth
+// parameters.
+// TODO: Add support for EngineID differentiation
```

Comment:
Is this something you planned to address before merging this PR?

Reply:
No this will have to be a different PR